### DISCUSSION TITLE
Is there roadmap/schedule for composer 2.1/2.2

### DISCUSSION THREAD

**Discussion by andypost** [composer/composer#9933] [Q&A / Support] [answered]
The 2.1-RC1 is out, milestone looks completed 92% https://github.com/composer/composer/milestone/13

Not clear when to start to check plugins compatibility checks and when to expect final releases

**Comment by stof** [comment, marked as answer]
> Not clear when to start to check plugins compatibility checks

now. that's what RC releases are about

**Comment by andypost** [comment]
While I used to type it [2.1](https://github.com/composer/composer/releases/tag/2.1.0) released) 

The question then is how to schedule future releases)))

**Comment by Seldaek** [comment]
We aren't super great at planning releases ahead of time.. It mostly happens when it happens. As such I think the best is to make sure you have at least one CI run against `composer/composer: ^2@dev` for plugins, so you can hopefully catch any failure early before it hits release.

If you use a phar file, you can use `composer self-update --snapshot` to test against our master branch.
